draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -2949,7 +2956,7 @@ Host: www.example.org
    connection-option in this field, remove any header or trailer field(s) from
    the message with the same name as the connection-option, and then
    remove the Connection header field itself (or replace it with the
-   intermediary's own connection options for the forwarded message).
+   intermediary's own control options for the forwarded message).
```

Comment:
@royfielding - was that change intentional?

Reply:
Yes, they are referred to as control options in the definition. A connection-option is the ABNF term.